### ISSUE TITLE
Pixiv infinite scroll seems to be non-functional

### ISSUE THREAD

**Issue by lioncash** [an-electric-sheep/userscripts#1]
This seems to have stopped working recently. Sorry for not having more detailed information on why.

**Comment by an-electric-sheep** [comment]
Yeah, I know, pixiv changed their layout and they're using dynamically generated content now which makes it more difficult to modify. I'm looking into it, but it make take a bit.
